gooddata-sdk/gooddata_sdk/catalog/workspace/service.py

```diff
@@ -806,6 +809,149 @@ def set_translated_texts(
                         if "description" in section["header"]:
                             section["header"]["description"] = translated.get(section["header"]["description"])
 
+    @staticmethod
+    def _add_target_tags(xliff_content: str, translate_func: Callable) -> bytes:
+        """Add target tags to the XLIFF content for translation purposes.
+
+        Args:
+            xliff_content (str): The XLIFF content as a string.
+            translate_func (Optional[Callable]):
+                A function that translates the source text. It can take an optional argument
+                `already_translated` and `old_translation` for updating existing translations.
+
+        Returns:
+            bytes: The modified XLIFF content with target tags, encoded as UTF-8.
+        """
+        ns = get_namespace_from_xml(xliff_content)
+
+        ET.register_namespace("", ns["ns"])
+        tree = ET.ElementTree(ET.fromstring(xliff_content))
+        root = tree.getroot()
+
+        # Segment is always parent of source/target - no need to find parents
+        for segment in root.findall(".//ns:segment", namespaces=ns):
+            source = segment.find("ns:source", namespaces=ns)
+            if source is not None and "".join(source.itertext()).strip():
```

Comment:
You reuse `"".join(source.itertext()).strip()` multiple times. Store it in a variable.